Sources/Networking/Responses/CustomerInfoResponse.swift

```diff
@@ -61,6 +61,9 @@ extension CustomerInfoResponse {
         @IgnoreDecodeErrors<PurchaseOwnershipType>
         var ownershipType: PurchaseOwnershipType
         var productPlanIdentifier: String?
+        #if ENABLE_PURCHASE_PARAMS
+        var metadata: [String: String]?
+        #endif
```

Comment:
Currently, the metadata information is only present in the `Subscription` object in the `customerInfo.subscriber.subscriptions` array. We'll expand this to expose it in other places.

Reply:
Hmm right... How are we going to handle one time purchases?